src/pages/bounties/[id]/[[...slug]].tsx

```diff
@@ -353,6 +337,8 @@ const BountySidebar = ({ bounty }: { bounty: BountyGetById }) => {
     addBenefactorUnitAmountMutation({ bountyId: bounty.id, unitAmount: amount });
   };
 
+  const meta = bounty.details as BountyDetailsSchema;
```

Comment:
We could probably do this casting at the server so it's available anywhere we call the endpoint